src/color/p5.Color.js

```diff
@@ -777,11 +769,10 @@ const colorPatterns = {
  * <div>
  * <code>
  * // todo
+ * //
+ * // describe('');
```

Comment:
@Zearin  could you open an issue for this missing reference?

Reply:
Does it matter that this is for a private function?  (It is not documented in the reference docs.)

I’m fine to open the issue if it should be added—just want to check about this point first.

Reply:
doesn't matter for reference but could be helpful for contributors to see an example

Reply:
👍

Reply:
Opened #5584 to address this one.